lib/core/Cargo.toml

```diff
@@ -25,8 +25,9 @@ lwk_signer = "0.5.1"
 # Switch back to published version once this PR is merged and included in release: https://github.com/Blockstream/lwk/pull/34 (ETA in v0.5.2)
 lwk_wollet = { git = "https://github.com/Blockstream/lwk", rev = "ffd793d0a1b1122c9bba7de23ccb73033eded98c" }
 #lwk_wollet = "0.5.1"
-rusqlite = { version = "0.31", features = ["backup", "bundled"] }
+rusqlite = { version = "0.29", features = ["backup", "bundled"] } # Same version as used in sdk-common
```

Comment:
Why does the sdk-common crate need rusqlite? I know its the FromSql/ToSql implementations, but I think it should the impartial and up to the application to implement these

Reply:
In principle I agree, but Rust doesn't allow you to implement traits for structs derived in another crate.

To simplify `sdk-common` later on, we could place `rusqlite` behind a feature flag, so callers who don't need it don't have to download it.

Reply:
The FromSQL ToSQL are helpers to read/write structs to the db using rusqlite.
We don't necessary have to use these helpers.
I think there are two points in the code where we use them:
1. sql_row_to_payment (for read)
2. insert_payment_external_info (for write)

I think it worth serializing manually to/from string there (serde_json::to_string, serde_json::from_str) and get rid of the rusqlite depenency.

Reply:
Done in https://github.com/breez/breez-sdk/pull/1019